src/PageContent/RequestPickup/RequestPickupWrapper.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import RecurringPickupRequest from './RecurringPickupRequest';
+import RequestTypeToggle from './RequestTypeToggle';
+import { DeliveryType } from '../../Enums';
+
+class RequestPickupWrapper extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            type: DeliveryType.EMERGENCY
+        };
+    }
+
+    toggle() {
+        if (this.state.type === DeliveryType.RECURRING) {
+            this.setState({ type: DeliveryType.EMERGENCY });
+        } else {
+            this.setState({ type: DeliveryType.RECURRING });
+        }
+    }
+
+    render() {
+        return (
+            <div>
+                <RequestTypeToggle
+                    type={this.state.type}
+                    toggle={this.toggle.bind(this)}
+                />
+                <RecurringPickupRequest
+                    account={this.props.account}
+                    donatingAgency={this.props.donatingAgency}
+                    type={this.state.type}
+                />
```

Comment:
I think we should have 2 separate UI components for `Recurring` and `Emergency` pickup requests, right now `RecurringPickupRequest` is way too big (and the component naming isn't accurate anymore). So I think we should have:
```
<RequestTypeToggle />
<PickupRequestForm />
```
Where `<PickupRequestForm />` should kinda be the current `<RecurringPickupRequest>` containing all the validation, create request, submit functions, etc. And in it's render functions, instead of having all the html there,
```
render() {
    return (
        {this.props.type === DeliveryType.RECURRING ?
            <RecurringPickupRequest />
        :
            <EmergencyPickupRequest />
        }
    );
}
```
`<RecurringPickupRequest>` contains the UI html code for the recurring form, and `<EmergencyPickupRequest>` contains the new form you added.